### PR TITLE
Docs: Add figure for navigating help resources

### PR DESCRIPTION
**Pull request by JessicaS11**
- [X] Addresses #8990 
- [x] User visible changes (including notable bug fixes) are documented in `whats-new.rst`

### CODE REVIEW

**Comment by max-sixty** [comment]
Very cool! Thanks @JessicaS11 

Couple of suggested changes (maybe others will disagree, I see a draft of this was reviewed in the discussion):
- Can we add a line directly from having a bug or type to PR? No need for folks to open an issue for something small.
- Possibly PRs should be inside the contributions box?
- Add LLMs as an option in "How do I use Xarray"?
- I actually don't think it's helpful to ping issues. PRs are waiting on maintainers, but issues need contributions not reminders, so I'd remove that arrow.
  - Pinging can just be "Gentle ping", no need to @ an individual (and not sure how they'd know who to @...)

**Comment by JessicaS11** [comment]
Readthedocs is failing with: `fatal: reference is not a tree` for the last commit. I'm assuming this is somehow caused by the way I added to the PR, but I'm also not sure how to fix it. To add to the PR after the precommit.ci commit, I ran ` git fetch upstream pull/9147/head:help-figup` and `git rebase db637420` before `git push origin help-fig:help-fig`. Is there another way to do this that won't upset readthedocs?

**Comment by keewis** [comment]
looks like that was just a temporary hickup: https://readthedocs.org/projects/xray/builds/24763397/

**Comment by JessicaS11** [comment]
> Couple of suggested changes (maybe others will disagree, I see a draft of this was reviewed in the discussion):
Thanks for sharing these - I'll commit once I've addressed them all.
>     * Can we add a line directly from having a bug or type to PR? No need for folks to open an issue for something small.
I'll update these
>     * Possibly PRs should be inside the contributions box?
Good call!
>     * Add LLMs as an option in "How do I use Xarray"?
Done!
>     * I actually don't think it's helpful to ping issues. PRs are waiting on maintainers, but issues need contributions not reminders, so I'd remove that arrow.
I guess I was thinking the case where the issue is a proposal that needs "approval" or feedback for the contributor to work on and get to the point of submitting a PR. Bug report issues aren't generally going to be "quiet" because maintainers are likely to be working on fixing them.
>       * Pinging can just be "Gentle ping", no need to @ an individual (and not sure how they'd know who to @...)
What is a "gentle ping" then? Simply posting a comment in the PR?

**Comment by max-sixty** [comment]
> What is a "gentle ping" then? Simply posting a comment in the PR?

Sorry, yes, I'm not sure the best wording, I was thinking `fab:fa-github Gentle ping!` might be a nice informal way of showing it, but maybe that's confusing, feel free to riff...

> I guess I was thinking the case where the issue is a proposal that needs "approval" or feedback for the contributor to work on and get to the point of submitting a PR. Bug report issues aren't generally going to be "quiet" because maintainers are likely to be working on fixing them.

IME it's quite rare that things need approval to get started, at least in projects like xarray. And there are almost 1000 issues so pinging for updates is somewhat disruptive.

So I would probably:
- add an arrow from the root to contributors' guide "I'd like to make a small change"
- drop the "my plan was approved"
- switch "I found a bug or typo" -> "I found a bug"

Is that OK? I don't mean to be wordsmithing. (It's actually quite interesting to have to make legible lots of implicit understanding of how to operate.)

**Comment by JessicaS11** [comment]
> Is that OK? I don't mean to be wordsmithing. (It's actually quite interesting to have to make legible lots of implicit understanding of how to operate.)

It's great - I completely agree on the making explicit the implicit being an interesting exercise.

I went with "Comment on your PR" - to me a "ping" means an @ mention or Slack/email message (basically explicitly contacting a group or individuals), which I'm sure is one of many definitions people have for it!

File: doc/getting-started-guide/index.rst
Comment (dcherian) [inline doc/getting-started-guide/index.rst:16]: Shall we move this to the top-level index.rst instead?

Comment (JessicaS11) [reply doc/getting-started-guide/index.rst:16]: Moved, but happy to put back within "Getting Started" if that's the overall preferred spot.

**Comment by max-sixty** [comment]
I think this is looking really good! Thanks @JessicaS11 !

**Review by max-sixty — APPROVED** [review]

**Comment by JessicaS11** [comment]
> I think this is looking really good! Thanks @JessicaS11 !

Thanks! I'm going to see if I can get it to render properly in the docs before merging (hence ongoing changes). Right now it's building successfully but showing up as text...

**Comment by JessicaS11** [comment]
Hey team - I'm hitting a wall here on how to get this rendered. Specifically, it appears the [sphinxcontrib.mermaid](https://github.com/mgaitan/sphinxcontrib-mermaid/tree/master) extension uses an older version of mermaid (10.2.0) than the [mermaid.live](https://mermaid.live/edit#pako:eNqNVmtv2kgU_SuT6YdSCWjCBghWtVU2JC0S6aIkUruL82GwL2aW8Yw7jwCN89_3jh9AAiutkTBmzrnPcy8800jFQAPaarVCabkVEJBv7IknzHKZkB9Ma7Yhd2CU0xGYUBbAUM6FWkULpi15GIaS4MWl1Wr6HUSkUiBWVdwT8lWtSMQkWQFZgMg-PwZB8NOB-Yae56kt2c6wBBrTkM5ZMGctNC2WM8V03HIGdB3Hg7NKcyZMSfLXnM08QQOL0W2FG6roEJIolQggX4rbx-pLY1m0bKkn0D4jcu8fyfUa_csE9k0UTtRMWXJpluRy9JGRMWIchk3GwLT05brFlARpjMe3H0L6-AETZfJ1npeTUWP6_2OuILC2IA1X8r0hMeKOmvbF3dpOuF242dY2N5Ez3sChixjPlI73ofj4NnXfBVNj_pzPeQTYWKfNW2Ai1AzIxJdPlcYQBEcDnrmkMb2DTKGKmIzJRKtMGa8SDcFh-15lNDIGJXTUbKRQiXzmS-F5M6WwwRnImnpVnnspYTm_OB4fDy_TpR73vU-cEDgNXr6WNCZ3_9XmGJ7etuJKpSlIS5QkG6wJmdwdr_EQnkBgvD64WwA_hkdd6KJw0wdU6tKQudJEgC2G1hmylGp1cnTQUtAJxFVtkBubViRYlnniX2VcZMVMhWvvYtxzFNUV9KTdpB_486_taiCt1u-53wWxIiOMsRb-57wc_r09UEBDusLC7aRvSEPwJZAhM0sse-5HqXawY43ex6SAGb-FUqXLpZMX72-jGWE2DoXHvBRz_3aAqM1hlinDOyMmZaiBckHkdSlKnnGzRLNs4S1VB9M_XGIKdW9l53N53NV163VPu69Oq1D_cSi5FZMWYh-NBYwChZRXQnhF-fQ29jlfE7sAf3aSH3rJdOEk3dTdR0FoXIpxXumghIKMjzV1RFbcLOrxwoEXcbvdrupZguuJIp92frghPx0Hm_tpeTW63mqmay7K05ghzMmetjAhIYJ3g-H19c1502AkSwjedU57VxeDA95ucirazc3l5Wl3S7v-rX9z1j-gzdR6n3LTvR4eULaEvZZXxNqe4HJ5bzf4y4MhMCfQIuJahv-CoHOa2WakhNJ18LRJseIp4zH-KD97CyHFxqUQ0gA_ViZC2iyPDEROc7sZ-6VRQoTCHRrSUL6gLYZb7n4jIxpY7aBJXRYzC0POUKcpxRUgDH6bMUmDZ7qmwXl7cDoY9HuD7qDTueie9Zp0Q4OzXq_d6XTOu_2Li27_vNN5adJfSqGBs4L8d_G59AAxx8V6W_6rKP5cNKlWLllU3l7-BQYEyRo) editor I used to create the flowchart (10.9.1). As a result, there are syntax errors. My understanding of the solution pathways are:
1. get the current flowchart code to work in an older version of Mermaid (not sure of an easy way to do this).
2. build the diagram and include it in the docs differently (e.g. using mermaid-cli), which I don't understand how things are built under the hood well enough to execute on my own (see also my attempt to get such a figure into the xarray-tutorial, though there was discussion of moving that to the docs as well: https://github.com/xarray-contrib/xarray-tutorial/pull/266)

Any help or insight is greatly appreciated!

**Comment by dcherian** [comment]
I found a setting to set a custom mermaid version: https://sphinxcontrib-mermaid-demo.readthedocs.io/en/latest/#config-values

**Comment by JessicaS11** [comment]
> I found a setting to set a custom mermaid version: https://sphinxcontrib-mermaid-demo.readthedocs.io/en/latest/#config-values

@dcherian Thank you!! Should I rerun CI before merging?

**Pull request merged**
